c2rust-analyze/src/rewrite/expr/mir_op.rs

```diff
@@ -51,6 +52,15 @@ pub enum RewriteKind {
     RemoveAsPtr,
     /// Replace &raw with & or &raw mut with &mut
     RawToRef { mutbl: bool },
+
+    /// Cast `&` to `*const` or `&mut` to `*mut`.
+    CastRefToRaw { mutbl: bool },
+    /// Cast `*const` to `*mut` or vice versa.  If `to_mutbl` is true, we are casting to `*mut`;
+    /// otherwise, we're casting to `*const`.
+    CastRawToRaw { to_mutbl: bool },
```

Comment:
Is this meant to be casts from `*{const,mut} T` to `*{const,mut} T`, or also between different types?  Could you include that in the docs?